wikidp/controllers/pages.py

```diff
@@ -19,7 +19,7 @@ def get_item_context(qid, with_claims=True):
     schemas = None
     if selected_item:
         options = request.args.get('options', default=0, type=str)
-        if type(options) is str:
+        if isinstance(options, str):
```

Comment:
Thank you for fixing these type checking clauses! This is more pythonic and is easier to read